src/vega_viewer/vega/specs/histogram.cljs

```diff
@@ -60,12 +65,24 @@
                        {:text
                         {:template
                          "{{ datum.data | number:'.3s'}}"}}})
-            spec))]
+            spec))
+        duration-chart (fn [spec]
```

Comment:
Writing it out like this makes sense if you expect needing to assoc other things on the spec, do you?

If not, you can rewrite this and remove all the duplication with 61-67, i.e. we can call this var something generic like `customize-spec` (instead of current `abbreviate-x-axis-tick-labels`, and push the conditional into the only line that's different (on line 67).

But, before doing that, we need to talk about the expected functionality. Because right now, if you have `abbreviate-x-axis-tick-labels?` and `custom-duration-chart?` as true, the first one is ignored since the second overrides the spec. Is that the desired behavior? If so, that needs to be made clearer to the user, maybe be having the user set an option, like `:x-axis-tick-labels` that they can set to either `:abbreviate`, `:duration`, or leaving it unset (this would replace my above comment about changing the var name).

@KipSigei what do you think? Let me know if you want to chat about this tomorrow. Another side note, the old code can be written more clearly by using `cond-> spec ...` instead of the `if .... (... spec ...) spec` patten